frontend/src/container/FormAlertChannels/Settings/Email.tsx

```diff
@@ -0,0 +1,50 @@
+import { Form, Input } from 'antd';
+import { Dispatch, SetStateAction } from 'react';
+import { useTranslation } from 'react-i18next';
+
+import { EmailChannel } from '../../CreateAlertChannels/config';
+
+const { TextArea } = Input;
+
+function EmailForm({ setSelectedConfig }: EmailFormProps): JSX.Element {
+	const { t } = useTranslation('channels');
+
+	const handleInputChange = (field: string) => (
+		event: React.ChangeEvent<HTMLInputElement | HTMLTextAreaElement>,
+	): void => {
+		setSelectedConfig((value) => ({
+			...value,
+			[field]: event.target.value,
+		}));
+	};
+
+	return (
+		<>
+			<Form.Item
```

Comment:
Need to add email validations to the input. We can use the Form component to maintain the state of the form.

Reply:
This can be more than one email. When there is more than one it will comma a separate list of emails. I don't think the default email validation works on this.